### ISSUE TITLE
Drag and drop option for the Support Reference section

### ISSUE THREAD

**Issue by elsylambert** [unfoldingWord/tc-create-app#76]
Drag and drop option for the Support Reference section

**Comment by birchamp** [comment]
In order to make this generic dropdowns should be automatic for fields that are in the filter.  The list should be populated the same way the list is populated for the filter.

**Comment by birchamp** [comment]
@Joel-C-Johnson test what's been used for language drop-down to see if it will allow for custom text.   Also add the link that of the one you found.

**Comment by Joel-C-Johnson** [comment]
> @Joel-C-Johnson test what's been used for language drop-down to see if it will allow for custom text. Also add the link that of the one you found.

@birchamp 
Yes, we may be able to modify the current language drop-down list and can allow the user to add the custom texts. 
here are my findings : 
- https://mui.com/material-ui/react-autocomplete/#creatable

also,

I have used this npm module which may help to handle a huge data and creates a drop-down list but it won't allow creating custom texts
- https://www.npmjs.com/package/react-virtualized-select

**Comment by mandolyte** [comment]
@Birch When a GL uses this dropdown, do you want them to see their TA repo content? or uW's?

**Comment by mandolyte** [comment]
@Birch If users enter custom text, won't that cause validation errors? What sort of text are they putting there? And when they do, how is reflected in the occurrence note itself?

**Comment by birchamp** [comment]
@mandolyte It looks like the filter is listing the Target's content, so let's use that. 

The user should be allowed to add custom text because they may not have used that specific figure of speech in the file yet.

**Comment by BincyJ** [comment]
@mandolyte @superdav42 can you work on this issue.

**Comment by elsylambert** [comment]
Looks good in translationCore Create - v1.5.0 build 154-e836e43. Able to select support Reference and type in custom text as well.
